# 08/May/2009
# Patch to fix the path to data files for Gentoo
# Not sending this to upstrem.
# neurogeek@gentoo.org

--- bin/lavennin.orig	2009-05-08 01:05:48.000000000 -0430
+++ bin/lavennin	2009-05-08 01:07:07.000000000 -0430
@@ -164,8 +164,8 @@
     $HOME = get_home_directory();
 }
 
-my $DATA_DIR = $HOME . "/data";
-
+#Gentoo Data Dir
+my $DATA_DIR = "/usr/share/suopuhe/data";
 
 # making the TMP dir...
 unless ( -e "$HOME/tmp" || $www ) {
